README.md

```diff
@@ -1,83 +1,115 @@
+> This Rails budget app is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. 
 
-## Built With
+[Loom Video Demo](https://www.loom.com/share/a7bb24188d6b414493864c0481e790d7)
 
-- Ruby
-- Ruby on Rails
-- Postgresql
-- Bootstrap
+### Deployment
 
-## Getting Started
+[Live Demo](https://cryptic-cove-58855.herokuapp.com/)
 
-**To get a local copy up and running follow these simple example steps:**
+This API is made of three models:
+- User model
+- Ability model
+- Category model
+- Expense_category model
+- Expense Model
 
-### Prerequisites
+The API has the following endpoints:
 
-- [Ruby](https://www.ruby-lang.org/en/)
-- [Rails](https://gorails.com/)
 
-### Setup
+allow user to authenticate and return a token
 
-- Make sure you have Ruby on Rails set up properly on your computer
+`POST /users/sign_in`
 
-``` sh
-  git git@github.com:vic778/rails-budget-app
-```
+allows user to create an account
 
-### Install
+`POST /users/sign_up`
+
+Edit user informations
+
+`PUT /users/edit`
+
+returns all the items
+
+`GET / categories`
+
+allows admin to create an item
+
+`POST /categories/new`
+
+returns a specific item
+
+`GET /categories/id`
+
+allows admin to add transaction to categorie
+
+`PUT /categories/id/expense/new`
+
+allows admin to get transaction of a categorie
+
+`PUT /categories/id/expense`
 
-```sh
-  bundle install
-```
+allows admin to delete an item
 
-### Run
+`DELETE /categories/id`
 
-```
-  rails db: create
-```
+## Built With
+
+- Rails
+- Ruby 
+- Heroku
 
-```
-  rails db:migrate
-```
+## Getting Started
 
-```
-  rails s
-```
+Here are the steps to follow in order to get this project on your local computer.
+
+### Prerequisites
 
-### Test
+`rails v7.0.2 +`
 
-```sh
-  rspec
-```
+`ruby v3.0.2 +`
 
-### Rubocop
+### Setup
+
+clone this repo by typing `git clone https://github.com/vic778/rails-budget-app`
+
+### Install
+
+install the dependencies by typing `bundle install`
 
-```sh
-    rubocop --color
-```
 ### Usage
 
-To start recipe app you need to first `signup`
+start the local server by running `rails s`
+
+### Testing
+
+run the tests by typing `bundle exec rspec`
+
 
-## Authors
+## Author
 
 👤 **Victor Barh**
 
 - GitHub: [@Vvic778](https://github.com/vic778)
 - Twitter: [@victoirBarh](https://twitter.com/)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/victoir-barh)
+
+
 ## 🤝 Contributing
 
-Contributions, issues, and feature requests are welcome!
+Contributions, issues and feature requests are welcome!
 
-- Feel free to check the [issues page](https://github.com/vic778/rails-budget-app/issues).
+Feel free to check the [issues page](issues/).
 
 ## Show your support
 
-- Give a ⭐️ if you like this project!
+Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Microverse for giving us this chance
\ No newline at end of file
+- Hat tip to anyone whose code was used !
+
```

Comment:
- [ ] [IMPORTANT] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).